apps/spawner/src/server.ts

```diff
@@ -129,8 +129,7 @@ export async function startAPIServer({ port }) {
           const doc = getMyYDoc({ repoId, token });
           const rootMap = doc.getMap("rootMap");
           const runtimeMap = rootMap.get("runtimeMap") as Y.Map<RuntimeInfo>;
-          runtimeMap.set(runtimeId, {});
-          // runtimeMap.delete(runtimeId)
+          runtimeMap.delete(runtimeId);
```

Comment:
This is a good change, I meant to use `runtimeMap.delete` but forgot to uncomment it. I've actually included this in my local code.